src/java.base/share/classes/java/io/DataInputStream.java

```diff
@@ -33,9 +33,10 @@
  * way. An application uses a data output stream to write data that
  * can later be read by a data input stream.
  * <p>
- * DataInputStream is not necessarily safe for multithreaded access.
- * Thread safety is optional and is the responsibility of users of
- * methods in this class.
+ * A DataInputStream is not safe for use by multiple concurrent
+ * threads. If a DataOutputStream is to be used by more than one
```

Comment:
Hi Andrew, just noticed a typo here: DataOutputStream
best regards,
-- daniel

Reply:
Yes, that was a comment on the previous iteration too. Also s/output/input/.